Objects/odictobject.c

```diff
@@ -1388,6 +1388,7 @@ odict_repr(PyODictObject *self)
 
 Done:
     Py_ReprLeave((PyObject *)self);
+    Py_XDECREF(dcopy);
```

Comment:
You can move it above the `Done` label and change `Py_XDECREF` to `Py_DECREF`.

Reply:
Thanks, it might be a slightly more readable! I will wait for all tests to finish and apply it :)